### ISSUE TITLE
Support for CiviCRM 4.7

### ISSUE THREAD

**Issue by keirlawson** [veda-consulting/uk.co.vedaconsulting.payment.ukdirectdebit#36]
Would be great to be able to use this on a CiviCRM website running 4.7

**Comment by TomCrawshaw** [comment]
Can anyone confirm (or otherwise) if this extension is working with Civi 4.7 under Drupal 7.43? I've tried installing and get fatal errors.

**Comment by verblet** [comment]
I've ran an upgrade test and it doesn't work for me. Billing block doesn't show.

**Comment by Kajakaran** [comment]
@verblet 
Can you provide what version of extensions(uk.co.vedaconsulting.payment.smartdebitdd
 and uk.co.vedaconsulting.payment.ukdirectdebit), CiviCRM and environment you have tested?

Thanks
Kajan
@veda-consulting

**Comment by Upperholme** [comment]
I've just enabled this extension on a 4.6.18 site running Drupal 7 as I'm looking to get GoCardless set up alongside Paypal Standard. With the Direct Debit extension enabled the billing block does not show on an event registration form using Paypal as the processor when pay later is selected, leading to validation errors and confusion.

**Comment by bhahumanists** [comment]
Yep, can confirm this on 4.7. The Billing Block template code looks like it's from 4.6...

**Comment by mattwire** [comment]
Please have a look at PR https://github.com/veda-consulting/uk.co.vedaconsulting.payment.ukdirectdebit/pull/55

**Comment by mattwire** [comment]
For CiviCRM 4.7+ please use the rewritten smartdebit extension here: https://github.com/mattwire/uk.co.vedaconsulting.smartdebit
